types/forge-viewer/index.d.ts

```diff
@@ -1838,6 +1844,61 @@ declare namespace Autodesk {
               restoreState(viewerState: object, filter?: object, immediate?: boolean): boolean;
             }
 
+            class ViewerStateObject {
```

Comment:
Since there already is a type called "ViewerState" I used "ViewerStateObject". To indicate that it is just a javascript object.